app/etl/item/content/parsers/body_content.rb

```diff
@@ -1,6 +1,9 @@
 class Item::Content::Parsers::BodyContent
   def parse(json)
-    json.dig("details", "body")
+    parsed = json.dig("details", "body")
+    parsed = parsed[0].dig('content') if parsed.present? && parsed.is_a?(Array)
```

Comment:
It looks like when the body is an array there can be multiple items with different mime types

I ran this query on dev:

```
publishing_api_development=> select foo.size, count(*) from (select json_array_length(json_extract_path(details, 'body')) as size from editions where json_extract_path(details, 'body', '0') is not null) foo group by foo.size;
 size | count
------+-------
    1 | 33809
    2 | 97092
```

An example of the second case is `/guidance/manual-of-patent-practice-mopp/section-14-the-application`

which has a body like this:

```json
[
{"content_type":"text/govspeak","content":"##Section 14: The application   \r\n\r\n###14.01\t\r\n\r\nThis section prescribes the manner of making an application, its form and content, the requirements which must be fulfilled by the contents, and provides for withdrawal of the application. ...."},
{"content_type":"text/html","content":"<h2 id=\"section-14-the-application\">Section 14: The application</h2>\n\n<h3 id=\"section\">14.01</h3>\n..."}
]
```

I think in that case we should be preferring one of them (text/html or text/govspeak) rather than just taking the first one.

Otherwise I don't think we can guarantee what we end up with and we might calculate metrics in an inconsistent way.

Reply:
Thanks @MatMoore - fixed in 1e74471dc0c6c9aafc1f5384fea42cbecf14d7f2